onnxruntime/core/providers/cuda/cuda_execution_provider.cc

```diff
@@ -1478,8 +1478,8 @@ CUDAExecutionProvider::GetCapability(const onnxruntime::GraphViewer& graph,
       std::vector<std::string> activations_supported{"sigmoid", "tanh", "sigmoid", "tanh"};
       not_supported = RNNNeedFallbackToCPU(node, activations_supported, node.OpType());
       force_inside = !not_supported;
-    } else if ("Conv" == node.OpType()) {
-      not_supported = ConvNeedFallbackToCPU(node);
+    } else if ("ConvTranspose" == node.OpType()) {
```

Comment:
No longer fall-back to CPU for asymetrically padded Conv nodes. But, a similar fallback logic is required for ConvTranspose which was missing and hence added it (until we can apply a similar logic to ConvTranspose as well).